src/examiner/examiner.js

```diff
@@ -65,3 +61,7 @@ function isLongTaskApiSupported(win) {
     'containerName' in win.TaskAttributionTiming.prototype
   );
 }
+
+if (isLongTaskApiSupported(self)) {
+  detectLongTasks(self);
+}
```

Comment:
Any reason this code needed to be moved?

Reply:
This looks like a strange rebase artifact. I'll revert and see if it breaks anything.

Reply:
It is actually a change required by build fixes for `esm` output.

```
src/examiner/examiner.js:17: ERROR - [JSC_REFERENCE_BEFORE_DECLARE_ERROR] Illegal variable reference before declaration: isLongTaskApiSupported
if (isLongTaskApiSupported(self)) {
    ^^^^^^^^^^^^^^^^^^^^^^
```